src/loki_alert_rules/kafka_rate_limit.rule

```diff
@@ -0,0 +1,8 @@
+alert: KafkaRateLimit
```

Comment:
Should we stick to the standard format as much as we can?

Reply:
Feel free to suggest any other name. It could just as easily be `TESTTESTTEST` without knowing what logs they actually care about, and the primary purpose is "check rule forwarding from the principal all the way back to Loki does what we expect"